urbansim/models/transition.py

```diff
@@ -94,8 +94,13 @@ def remove_rows(data, nrows, accounting_column=None):
     unit_check = data[accounting_column].sum() if accounting_column else len(data)
```

Comment:
With this `unit_check` variable, don't the different interpretations that `nrows` could take already get accounted for?   So if accounting column is not None, unit_check is data[accounting_column].sum(),   and if accounting column is None, then unit_check is len(data).   Given that, doesn't the existing code already the accounting_column case?